test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -227,19 +227,29 @@ test_pass_indented_on_contents_default:
         s.b
     FROM r
     JOIN s
-        ON r.a = s.a
```

Comment:
Since it would all fit on one line, could we generate:
```
        ON r.a = s.a AND true
```
?

Reply:
See comment on currently not _removing_ newlines. Let's handle that as part of "long lines".